src/uws_client.c

```diff
@@ -1730,26 +1736,15 @@ int uws_client_close_async(UWS_CLIENT_HANDLE uws_client, ON_WS_CLOSE_COMPLETE on
 
```

Comment:
This could potentially overwrite the USW_STATE_CLOSING_SENDING_CLOSE or UWS_STATE_CLOSING_WAIT_FOR_CLOSE, does it matter?

Reply:
I think the behavior is in line with the description "uws_client_close_async` shall close the uws instance connection if an open action is either pending or has completed successfully (if the IO is open)." The change is necessary to be able to force close if waiting for gracefully closing the connection (with the handshake) fails.